libs/pika/async_mpi/tests/unit/CMakeLists.txt

```diff
@@ -40,7 +37,7 @@ foreach(test ${tests})
     FOLDER "Tests/Unit/Modules/AsyncMPI"
   )
 
-  foreach(polling_mode RANGE 0 63 1)
+  foreach(polling_mode RANGE 0 ${MPIX_CONTINUATIONS_LOOP_COUNT} 1)
```

Comment:
```suggestion
  foreach(polling_mode RANGE 0 ${PIKA_MPIX_CONTINUATIONS_NUM_METHODS} 1)
```
?

Reply:
oops didn't see this until now, but I already renamed the loop counter to `PIKA_MPI_MODES_LOOP_COUNT`

Reply:
Ok. I thought we can leave out `LOOP_COUNT` since that signals how it's used, rather than what it represents, but let's stick this for now then.